apps/asap-server/src/controllers/groups.ts

```diff
@@ -58,13 +58,17 @@ export interface ResponseFetchGroups {
   };
 }
 
-type FetchOptions = {
-  take: number;
-  skip: number;
+export type FetchOptions = {
```

Comment:
For now we can keep this here, but eventually we need to move it into a shared file since it will be required by all other controllers